sct-commons/src/main/java/org/lfenergy/compas/sct/commons/dto/DaTypeName.java

```diff
@@ -116,6 +114,6 @@ public void merge(DaTypeName daName) {
         bType = daName.bType;
         type = daName.type;
         valImport = daName.valImport;
-        daiValues.putAll(daiValues);
+        daiValues.putAll(daName.daiValues);
```

Comment:
> is this code covered by test ?
Unfortunately no, otherwise it would have spotted that `daiValues.putAll(daiValues);` does nothing.

> By the way the method "merge" does not seemed to be used, is it used outside of the project ?
I don't use it. It is just something that my IDE showed me.

As It is not part of this issue correction, I suggest cancelling the change and creating an issue for that. Is that OK ?

Reply:
It is out of scope of the issue. I don't want to deal with testing it. I am cancelling the change and leave it as is.